napari/util/misc.py

```diff
@@ -214,38 +150,8 @@ def __str__(self):
         # render_kw_only_separator = True
         for param in self.parameters.values():
             formatted = str(param)
-
-            # kind = param.kind
-
-            # if kind == inspect._POSITIONAL_ONLY:
```

Comment:
@kne42 , I think you may have written `CallSignature()`. Is it okay to delete these commented out sections? I am happy to put them back in if we're saving them for later.

Reply:
I put that there because it's a modified version of `inspect.Signature` but it should be ok to delete I think

Reply:
Got it. Thanks!